### PR TITLE
Adding 20190704 meeting

### PR DESCRIPTION
**Pull request by teresanunez**

### CODE REVIEW

**Comment by teresanunez** [comment]
I have added some basic agenda. Please feel free to add more points there. Thanks.

File: 20190704/AGENDA.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Sardana Follow-up Meeting - 2019/07/04
+
+To be held on 2019/07/04 at 14:00
+
+## Agenda
+
+1. Presentation - any idea?
```
Comment (reszelaz) [inline 20190704/AGENDA.md]: IMO the more time we dedicate to SEP16 the better. I would for example discuss in detail the cookiecutter template that @amilan proposed in the last meeting.

File: 20190704/AGENDA.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      
+2. Urgent user problems/issues - Round table
+3. Overview of current developments / PR distribution
+    * SEP16 ([#726](https://github.com/sardana-org/sardana/pull/726))
+    * Migration to python3
+    * 'General condition' feature
+4. Jul19 release summary
+5. Any other business?
```
Comment (reszelaz) [inline 20190704/AGENDA.md]: * ICALEPCS2019 Sardana workshop?
* [Questionnaire](https://github.com/sardana-org/sardana-followup/issues/24)

**Review by reszelaz — APPROVED** [review]
LGTM, just some minor suggestions.

**Review by amilan — APPROVED** [review]

**Comment by reszelaz** [comment]
@teresanunez I think the aganda won't change. Could you please also sent a late email to the mailing list about today's meeting. Many thanks!

**Pull request merged**